programmers/소수-찾기.js

```diff
@@ -0,0 +1,41 @@
+function combine(splitedNums, N) {
+    let result = [];
```

Comment:
재할당 되지 않는 변수는 const사용하기
```suggestion
    const result = [];
```